externs/custom-elements.js

```diff
@@ -14,13 +14,13 @@ const CustomElementState = {
 /**
  * @typedef {{
  *  localName: string,
- *  constructor: !Function,
+ *  constructorFunction: !Function,
```

Comment:
Closure did not like that this variable was called `constructor` and simply removed it from all type checks. Therefore, if you would create an object that would adhere to this interface, Closure would report the `constructor` should not be there, as the `constructor` of an object is:

```js
> ({}).constructor
< ƒ Object() { [native code] }
```

By renaming to `constructorFunction`, we can circumvent this.